test/test.js

```diff
@@ -294,8 +285,7 @@ describe('sourcemap-concat', function() {
     });
     builder = new broccoli.Builder(node);
     return builder.build().then(function(result) {
-      expectFile('nothing.js').in(result);
-      expectFile('nothing.map').in(result);
+      expectValidSourcemap('nothing.js').in(result);
```

Comment:
maybe we should keep using `expectFile()` here and add a code comment about why `expectValidSourcemap()` is not used